### PR TITLE
add support to delete connection owner via CLI

### PR DESCRIPTION
**Pull request by leogermani**
#### Changes proposed in this Pull Request:

* Adds support to delete connection owner via jetpack disconnect CLI command

#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* Set up a full connection
* run `wp jetpack disconnect user 1` (where `1` is your admin/connection owner user id)
* Make sure you get a error message with the user login and email
* Make sure the error message knows it is the connection owner and suggests you to use `--force`
* run `wp jetpack disconnect user 1 --force`
* Make sure it succeeds
* Visit your dashboard and confirm you're now in site-only connection mode
* try to delete the user again and make sure it fails and the error message says the user is not connected

### CODE REVIEW

File: projects/plugins/jetpack/class.jetpack-cli.php
Comment (kraftbj) [inline projects/plugins/jetpack/class.jetpack-cli.php:240]: Is there a better way to error? Could we detect if it is the connection owner and state that instead?

Comment (leogermani) [reply projects/plugins/jetpack/class.jetpack-cli.php:240]: Sure there is. I did this quickly because I was focused on some last minute user-less bugs. I'll mark this as in progress and wrap it up properly.

Comment (leogermani) [reply projects/plugins/jetpack/class.jetpack-cli.php]: Done and updated the test instructions

**Review by jeherve — CHANGES_REQUESTED** [review]
This looks good to me. I only have a tiny typo nitpick that I'll commit right now to unblock you.

**Comment by leogermani** [comment]
Thanks @jeherve !

**Review by samiff — APPROVED** [review]

**Pull request merged**
